site/content/en/docs/tutorials/kubevirt.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "KubeVirt support"
+linkTitle: "KubeVirt support"
+weight: 1
+date: 2020-05-26
+description: >
+  Using KubeVirt with minikube
+---
+
+## Prerequisites
+
+- Linux
+- Nested Virtualization enabled
+- kvm2 driver
+
+### Enable KubeVirt on minikube
+Minikube can be started with default values and those will be enough to run a quick example, that being said, if you can spare a few more GiBs of RAM (by default it uses 2GiB), it’ll allow you to experiment further.
+
+We’ll create a profile for KubeVirt so it gets its own settings without interfering what any configuration you might have already, let’s start by increasing the default memory to 4GiB:
+
+```shell script
+minikube config -p kubevirt set memory 4096
+minikube config -p kubevirt set vm-driver kvm2
+minikube start -p kubevirt
+```
+
+To enable this addon, simply run:
+```shell script
+minikube addons enable kubevirt-provisioner
+```
+
+You will then need to check if your VM’s CPU supports virtualization extensions, you can check by executing the following command:
+```shell script
+minikube ssh -p kubevirt "egrep 'svm|vmx' /proc/cpuinfo"
+```
+If the command does not generate any output, create the following ConfigMap so that KubeVirt uses emulation mode, otherwise skip to the next section:
+```shell script
+kubectl create configmap kubevirt-config -n kubevirt --from-literal debug.useEmulation=true
```

Comment:
As discussed:

We can consider doing this with two options:

kubevirt -- regular
kubevirt -with-emmulation -- if emualation is needed to workaround missing kvm